src/OpenApi/test/OpenApiGeneratorTests.cs

```diff
@@ -82,9 +82,9 @@ public void AddsRequestFormatFromMetadata()
     {
         static void AssertCustomRequestFormat(OpenApiOperation operation)
         {
-            var request = Assert.Single(operation.Parameters);
-            var content = Assert.Single(request.Content);
-            Assert.Equal("application/custom", content.Key);
+            Assert.Empty(operation.Parameters);
```

Comment:
Supr nit: for additional confidence here what do you think about adding a `Assert.Empty(parameter)` where applicable if we are asserting on `operation.RequestBody`. For example, like here:

https://github.com/dotnet/aspnetcore/blob/5249c1fcc2a923494ef0082d99540eed4ca49d0c/src/OpenApi/test/OpenApiGeneratorTests.cs#L727